src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -19099,6 +19099,810 @@ public void AppendFormatted<T>(T hole, int alignment = 0, string format = null)
             Diagnostic(ErrorCode.ERR_InterpolatedStringHandlerMethodReturnMalformed, "{f2()}").WithArguments("?.()").WithLocation(1, 21));
     }
 
+    [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/78137")]
+    [CombinatorialData]
+    public void InterpolationHandler_ReceiverParameter_Identity(bool useMetadataRef)
```

Comment:
>InterpolationHandler_ReceiverParameter_Identity [](http://example.com/codeflow?start=16&length=47)

I think we can benefit from asserting IL for unique enough scenarios. In particular I am interested in understanding and have a baseline for receiver capturing. For example, when we have a by value struct receiver, it is going to be captured by reference or by value? Are ByRef locals going to be involved? If so, could this block some `async` scenarios, erc.